bookkeeper-server/src/test/java/org/apache/bookkeeper/client/api/BookKeeperApiTest.java

```diff
@@ -38,6 +43,9 @@
 import org.apache.bookkeeper.client.BKException.BKNoSuchLedgerExistsException;
 import org.apache.bookkeeper.client.BKException.BKUnauthorizedAccessException;
 import org.apache.bookkeeper.client.MockBookKeeperTestCase;
+import org.apache.bookkeeper.util.LoggerOutput;
+import org.apache.log4j.spi.LoggingEvent;
```

Comment:
don't use log4j's API directly, only slf4j

Reply:
It's pretty tricky here, as `slf4j` doesn't have appenders, and the test is done via capturing the appended `LoggingEvents`, at the `log4j` level...

Reply:
@dlg99: pushed an update bfe22f3 addressing this as described above